frontend-web/webclient/app/Project/Subprojects.tsx

```diff
@@ -266,6 +279,12 @@ const Subprojects: React.FunctionComponent = () => {
                                         quotaInTotal={isQuotaSupported(w.wallet.paysFor) ?
                                             quota.data.quotaInTotal : undefined
                                         }
+                                        usage={isQuotaSupported(w.wallet.paysFor) ?
+                                            quota.data.quotaUsed : undefined
+                                        }
+                                        remainingQuota={isQuotaSupported(w.wallet.paysFor) ?
+                                            quota.data.quotaInBytes : undefined
+                                        }
```

Comment:
The same computation is done here three times. Moving the call above and calculating it there would be a bit better.